README.md

```diff
@@ -3,7 +3,6 @@
 [![NPM version](https://img.shields.io/npm/v/@brightspace-ui/intl.svg)](https://www.npmjs.org/package/@brightspace-ui/intl)
 [![NPM downloads](https://img.shields.io/npm/dt/@brightspace-ui/intl.svg)](https://www.npmjs.com/package/@brightspace-ui/intl)
 [![Dependabot badge](https://flat.badgen.net/dependabot/BrightspaceUI/intl?icon=dependabot)](https://app.dependabot.com/)
-[![Build status](https://travis-ci.com/BrightspaceUI/intl.svg?branch=master)](https://travis-ci.com/BrightspaceUI/intl)
```

Comment:
Minor: We can add a Github Action status badge if we want: https://docs.github.com/en/free-pro-team@latest/actions/managing-workflow-runs/adding-a-workflow-status-badge

I'm indifferent - the status was never the useful part for me, but the link to Travis was.  Since now we don't need to leave the repo anymore... 🤷

Reply:
Good call. I removed the Dependabot badge as it's actually broken and GitHub knows it.